superset-frontend/babel.config.js

```diff
@@ -74,7 +78,6 @@ module.exports = {
             targets: { node: 'current' },
           },
         ],
-        ['@emotion/babel-preset-css-prop'],
```

Comment:
We don't need this any more?

Reply:
Yes, this is included in `@emotion/babel-plugin` already for new JSX runtime ([source](https://www.npmjs.com/package/@emotion/babel-preset-css-prop))

I already added it [here](https://github.com/apache/superset/pull/28571/files#diff-5cc6b0da6915b09d0c2d20aa46e0196b1daced2e21e726f404d31ec4579f97b5R59) :D